charts/kubermatic/values.yaml

```diff
@@ -268,12 +268,19 @@ kubermatic:
 
   # You can override the default containers used for managing user cluster backups
   # using these three options. If they are left empty, the default containers from
-  # the static/ directory will be used for the store and cleanup containers, and
-  # the delete container will be disabled
+  # the static/ directory will be used for the store and cleanup containers.
   # To disable backups, configure containers that just run /bin/true, for example.
   storeContainer: null
   deleteContainer: null
   cleanupContainer: null
+  # Settings for the the controllers for EtcdBackupConfig and EtcdRestore CRDs,
+  # which allow dynamically creating multiple arbitrary backup schedules for each
+  # cluster, as well as restoring a cluster's etcd from any of the created backups
+  etcdBackupsRestores:
+    enabled: false
+    s3:
+      endpoint: minio.minio.svc.cluster.local:9000
+      bucket: kubermatic-etcd-backups
```

Comment:
I don't think this is supported anymore. We moved to using the kubermatic-installer now, which uses the [`KubermaticConfiguration`](https://github.com/kubermatic/kubermatic/blob/master/pkg/crd/operator/v1alpha1/configuration.go) CRD and runs the operator with it. You probably should add the needed configuration into the CRD and update the operator chart instead to reflect the changes.

Reply:
@multi-io Is this resolved? I think this is the only remaining point in the PR.

Reply:
Sorry I overlooked that. Will work on it asap. I may need some help because we're not using the operator currently.